features/steps/core.py

```diff
@@ -66,6 +67,25 @@ def set_config(context, config_file):
             cf.write("version: {}".format(__version__))
 
 
+@when('we open the editor and enter')
+@when('we open the editor and enter {text}')
```

Comment:
This should fix it 😄 
```suggestion
@when('we open the editor and enter "{text}"')
```

Reply:
> I hate computers sometimes.

This is all I have to say about that.

Reply:
That makes behave think it's undefined for some reason.

```python3
You can implement step definitions for undefined steps with these snippets:

@when(u'we open the editor and enter ""')
def step_impl(context):
    raise NotImplementedError(u'STEP: When we open the editor and enter ""')
```